### PR TITLE
chore: [IAI-235] Add `Pictogram` and `SectionPictogram` components

### PR DESCRIPTION
**Pull request by dmnplb**
### ⚠️ This PR depends on #4101 
This PR adds `Pictogram` and `SectionPictogram` components with the goal of giving a tidier way to insert vector assets in the application. `Pictogram` component includes all the decorative pictograms, `SectionPictogram` all the pictograms associated to sections (Messages, Profile, Settings, etc…).

From now on, you can insert a pictogram in the following way:
```jsx
{/* Insert the pictogram Umbrella with default
120 × 120 size and `aqua` default color */}
<Pictogram name="umbrella" />
``` 

### Preview
| `SectionPictogram` | `Pictogram` |
| - | - |
| Simulator Screen Shot - iPhone 13 - 2022-08-09 at 14 44 04 | Simulator Screen Shot - iPhone 13 - 2022-08-09 at 14 44 23 |

## List of changes proposed in this pull request
- Add new `Pictogram` component with relative types
- Add new `SectionPictogram` component with relative types
- Add the local `README` to give proper documentation

## How to test
- Start the IO app with `yarn run-ios`
- Login to the app
- Go to the _Profile → UI Showroom_

### CODE REVIEW

File: ts/components/core/pictograms/Pictogram.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  pin: Pin,
+  timeout: TimeoutPictogram,
+  uploadFile: UploadFilePictogram,
+  hourglass: Hourglass,
+  teaBreak: TeaBreak,
+  sms: SmsPictogram,
+  condom: Condom,
+  heart: Heart,
+  completed: Completed
+};
```
Comment (cloudify) [inline ts/components/core/pictograms/Pictogram.tsx:60]:
```suggestion
} as const;
```

This will make the keyof work correctly

File: ts/components/core/pictograms/Pictogram.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  color = "aqua",
+  size = 120,
+  ...props
+}: IOPictogramsProps) => {
+  const PictogramElement = IOPictograms[name];
+  return (
+    <PictogramElement
+      {...props}
+      size={size}
+      color={IOColors[color] as IOColorType}
```
Comment (cloudify) [inline ts/components/core/pictograms/Pictogram.tsx:86]: If you need casting here, you probably have a problem in the mapped type

File: ts/components/core/pictograms/SectionPictogram.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+export const IOSectionPictograms = {
+  messages: SectionPictogramMessages,
+  payments: SectionPictogramPayments,
+  documents: SectionPictogramDocuments,
+  services: SectionPictogramServices,
+  profile: SectionPictogramProfile,
+  settings: SectionPictogramSettings,
+  smile: SectionPictogramSmile
+};
```
Comment (cloudify) [inline ts/components/core/pictograms/SectionPictogram.tsx:22]:
```suggestion
} as const;
```

File: ts/components/core/pictograms/Pictogram.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+export type SVGPictogramProps = {
+  size: number | "100%" | undefined;
+  color?: IOColorType;
+};
+
+const Pictogram = ({
+  name,
+  color = "aqua",
+  size = 120,
+  ...props
```
Comment (cloudify) [inline ts/components/core/pictograms/Pictogram.tsx:79]: why is this needed? There are no other props other than the ones you already declared here

File: ts/components/core/pictograms/Pictogram.tsx (hunk 1 above)
Comment (dmnplb) [reply ts/components/core/pictograms/Pictogram.tsx:60]: What was the problem? Never got issues with `keyof` applied to the `IOPictograms` object

File: ts/components/core/pictograms/Pictogram.tsx (hunk 2 above)
Comment (dmnplb) [reply ts/components/core/pictograms/Pictogram.tsx:86]: All the Pictograms SVG files (contained in `svg` folder) are using the following type:
```jsx
export type SVGPictogramProps = {
  size: number | "100%";
  color: IOColorType;
};
```
If I try to set that prop without casting, I have the following error:
```ts
Type 'string' is not assignable to type '"white" | "greyUltraLight" | "greyLight" | "bluegreyLight" | "grey" | "milderGray" | "bluegrey" | .. 23 more ... ts (2322)
Pictogram.tsx(72, 3): The expected type comes from property 'color' which is declared here on type 'IntrinsicAttributes & SVGPictogramProps'
```

File: ts/components/core/pictograms/Pictogram.tsx (hunk 4 above)
Comment (dmnplb) [reply ts/components/core/pictograms/Pictogram.tsx:79]: There could be some props related to accessibility, for example. So I thought it could be better to make the component less rigid.

Comment (cloudify) [reply ts/components/core/pictograms/Pictogram.tsx:80]: in this case, the type of `props` will always be an empty object as there are no properties left after the destructuring, so since it extra / unneeded code it's a kind of anti pattern to leave it

**Review by francescopersico — APPROVED** [review]

**Review by francescopersico — APPROVED** [review]
